### PR TITLE
NaiveBayes 0.3.1 and Boltzmann 0.5.0

### PR DESCRIPTION
**Pull request by dfdx**

### CODE REVIEW

File: NaiveBayes/versions/0.3.1/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+julia 0.4
+Distributions 0.6+
```
Comment (tkelman) [inline NaiveBayes/versions/0.3.1/requires]: best to use actual version numbers here, the + isn't all that useful as a lower bound

Comment (dfdx) [inline NaiveBayes/versions/0.3.1/requires]: Ah, leftover from the times I didn't really understand version specs. Should I update and re-submit committed tags or is it ok to fix it in master only?

Comment (tkelman) [inline NaiveBayes/versions/0.3.1/requires]: can fix it in-place just in this metadata branch, as long as the same change gets made on master prior to the next tag

Comment (dfdx) [inline NaiveBayes/versions/0.3.1/requires]: fixed, thanks for the tip

**Pull request merged**